libdevice/cmake/modules/SYCLLibdevice.cmake

```diff
@@ -91,7 +91,7 @@ function(add_devicelib_obj obj_filename)
 
   set(devicelib-obj-file-new-offload ${obj_new_offload_binary_dir}/${obj_filename}.${new-offload-lib-suffix})
   add_custom_command(OUTPUT ${devicelib-obj-file-new-offload}
-                     COMMAND ${clang} -fsycl -c --offload-new-driver
+                     COMMAND ${clang} -fsycl -c --offload-new-driver -foffload-lto=thin
```

Comment:
Will this impact the use of these library files in a regular flow (i.e full LTO)?

Thanks

Reply:
I haven't tried but it should not, in that case we read in the full BC file to link. The extra information in the BC file provided added by the thinLTO infrastructure allows linking without loading the full BC file.

Reply:
great. Thanks much